projects/storefrontapp-e2e-cypress/cypress/helpers/order-cancellations-returns.ts

```diff
@@ -0,0 +1,33 @@
+export function checkTabs() {
+  cy.get('cx-tab-paragraph-container > h3')
+    .eq(1)
+    .should('contain', 'RETURNS')
+    .click();
+
+  cy.get('cx-tab-paragraph-container > h3')
+    .first()
+    .should('contain', 'ALL ORDERS')
+    .click();
+}
+
+export function cancelOrder() {
+  cy.get('cx-order-history a.cx-order-history-value')
+    .contains('In Process')
+    .click({ force: true });
+
```

Comment:
when clicking in process, you are navigating to another page technically. 

Try waiting for the page to load using the `waitForPage` function from `checkout-flow.ts`